### DISCUSSION TITLE
Will the T10 Thermostat be supported?

### DISCUSSION THREAD

**Discussion by robboz4** [d0n4v4nb3ck3r/homebridge-honeywell-home#586] [Q&A]
HI!

First ofall  thank you for making a plugging for the Honeywell Thermostats. We just had our AC system updated as it was 20 years old and I wanted to connect to Homebridge.

I installed the plugin and don't get the current temperature reflected in Homebridge accurately. I then noticed that the T10 was not listed in supported Thermostats so I'm assuming that is the issue. 
I see this in thelog:

[11/3/2021, 9:38:57 AM] [Homebridge UI] [homebridge-honeywell-home] Incoming Request: Start Honeywell Login Server
[11/3/2021, 9:39:04 AM] [RPi] 127.0.0.1: set Load from 0.68 to 0.9
[11/3/2021, 9:39:17 AM] [Homebridge UI] [homebridge-honeywell-home] Terminating child process...
[11/3/2021, 9:39:17 AM] [Homebridge UI] [homebridge-honeywell-home] Child process ended

So it looks like it is trying to get information. Also in the plugin config it does list the thermostat type as "undefined" , but reports the firmware version. However the config on the icon  reports it as a T9-T10. It also accurately reports the threshold setting we have set to call for heat.

Is there any other setting I need to adjust or wait for support for the T-10.

Thanks once again!

**Comment by donavanbecker** [comment]
Please open a issue and follow the template as this device should be similar to the T9. I will need more info which the issue template will get me if you fill it all in.
